oneview/resource_ethernet_network.go

```diff
@@ -202,6 +254,33 @@ func resourceEthernetNetworkUpdate(d *schema.ResourceData, meta interface{}) err
 		return fmt.Errorf("Initial scope uri can not be updated")
 	}
 
+	if d.HasChange("bandwidth") {
+		rawVal := d.Get("bandwidth")
+		bandwidthVal := rawVal.([]interface{})
+		for _, bandwidth := range bandwidthVal {
+			rawBandwidth := bandwidth.(map[string]interface{})
+			conTempURI := utils.NewNstring(d.Get("connection_template_uri").(string))
+			// get connection template by uri
+			conTemp, err := config.ovClient.GetConnectionTemplateByURI(conTempURI)
+			if err != nil {
+				return fmt.Errorf("unable to retrieve connection template: %s", err)
+			}
+			// filter URI
+			id := strings.Split(conTempURI.String(), "/")[3]
+			// update the con_temp with required bandwidth
+			BandwidthOptions := ov.BandwidthType{
+				MaximumBandwidth: rawBandwidth["maximum_bandwidth"].(int),
+				TypicalBandwidth: rawBandwidth["typical_bandwidth"].(int),
+			}
+			conTemp.Bandwidth = BandwidthOptions
+			conTemp, err = config.ovClient.UpdateConnectionTemplate(id, conTemp)
+			if err != nil {
+				return fmt.Errorf("unable to update bandwidth: %s", err)
+			}
+		}
+
+	}
+
 	err := config.ovClient.UpdateEthernetNetwork(newENet)
```

Comment:
here I have one interesting question: as a user i only update bandwidth nothing else so as per this code connection template update will happen and network update will happen but network update is totally unnecessary here. Just food for thought

Reply:
As this PR is focused on the bandwidth attribute, that is totally a different discussion. Maybe we can park this as a future enhancement discussion?